server/src/main/resources/db/changelog/changes/05_thesis_presentations.sql

```diff
@@ -0,0 +1,4 @@
+--liquibase formatted sql
+
+--changeset emilius:05-thesis-presentations-1
+ALTER TABLE thesis_presentations ADD visibility TEXT NOT NULL DEFAULT 'PRIVATE';
```

Comment:
The text is not limited to any length, I think the other migrations had upper bounds for text length, right?

Reply:
I decided against limiting text length in the database as I dont see any benefit. Under the hood TEXT and VARCHAR(1000) use the same type but just adds a limit check. Modifying accepted string length should not require database modifications and I do that in the backend. I dont have any string limits in the database right now

Reply:
Yup sounds good, thanks for the reasoning :) In MySQL, all `TEXT` datatypes are dynamically adjusted in memory utilization so this sounds like a good design 👍

Reply:
Its actually postgres in that case but for postgres TEXT is basically always the better type than character varying